docs/organization/1-3-working.md

```diff
@@ -50,8 +50,8 @@ Temos dois tipos de apontamento na Magrathea. Um é o ponto padrão, requerido p
 
 Utilizamos para controle de batidas um sistema interno, que está em desenvolvimento. Ele é uma aplicação PWA o que possibilita a utilização como aplicativo no celular.
 
-Link para o app: http://ponto-app.magrathealabs.io/
-Link para o admin: http://ponto-admin.magrathealabs.io/
+Link para o app: https://ponto-app.magrathealabs.io/
```

Comment:
Vi agora que esse está "Offline for maintenance", mas suponho que seja um probleminha temporário e que normalmente funcionaria em HTTPS

Reply:
**acho** que foi unificado, admin e app são somente esse aqui agora https://ponto-admin.magrathealabs.io/

cc @rhogeranacleto

Reply:
inclusive o daleponto n ta com redirect pra https ne? se eu botar http ele continua http